content/pages/meetup/index.rst

```diff
@@ -6,8 +6,18 @@ meetup
 :url: meetup
 :save_as: meetup/index.html
 
-meetup 2023
+Meetup 2023
 ==============
 
+* `Sciwork Meetup 22nd March 2023 in NYCU
+  <{filename}2023/0322.rst>`__
+
 * `Meetup 2023 March at NYCU
-  <{filename}2023/03-nycu.rst>`__
\ No newline at end of file
+  <{filename}2023/03-nycu.rst>`__
```

Comment:
I think it will be more organized to add the agenda to this existing file, instead of creating the new `0322.rst`.